dali/operators/random/rng_base.h

```diff
@@ -121,6 +121,15 @@ class RNGBase : public OperatorWithRng<Backend> {
    */
   void AcquireArgs(const OpSpec &spec, const Workspace &ws, int nsamples) {}
 
+  /**
+   * @brief If the output shape is non copied from `shape` argument or `__shape_like` input,
```

Comment:
Nitpick, maybe not worth another `!build`
```suggestion
   * @brief If the output shape is not copied from `shape` argument or `__shape_like` input,
```